src/styles/cards.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+
+  height: 408px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  background-color: $primary-bg-color;
```

Comment:
card background color should be `#ffffff` not `#f3f3f3`.

Yours:
![image](https://github.com/mate-academy/layout_product-cards/assets/52002856/a5961d4f-7451-4b24-b660-84e966448680)

Figma's:
![image](https://github.com/mate-academy/layout_product-cards/assets/52002856/0f162fdd-edff-4e06-a39b-4cde607b42d2)